lang/js/index.md

```diff
@@ -36,11 +36,25 @@ Variables, methods and property names will use `camelCase` style formatting.  On
 
 Functions are named only if they are not supposed to be overridden.  Anonymous functions do not need a name and shouldn't get one and should either be passed directly or assigned to a variable.
 
-When declaring multiple variables, they should be declared alphabetically and any symbols at the front of variable names should be ignored. Where there is a conflict between two matching variable names, capitalised variable names should be listed first; think carefully before you name two variables similarly because that typically indicates a problem.
+When declaring multiple variables, they should be declared alphabetically and any symbols at the front of variable names should come before ones without. This is enforced with the ESLint configuration we use. Where there is a conflict between two matching variable names, capitalized variable names should be listed first; think carefully before you name two variables similarly because that typically indicates a problem.
 
 It is typical, when using jQuery, to name your jQuery-ized variables with a `$` in front, like `$elements = $('input');`.
 
 
+    // Alphabetically Correct
+    function someFunction () {
+        var $car, bus, Car, car;
+
+        // Work here
+    }
+
+    // Alphabetically Incorrect
```

Comment:
The guide tries to never show bad examples because two examples means it makes the user think more and they might glam on to the wrong example and say that the style guide says it is ok.  To prevent confusion, I think this should be removed.